votrfront/js/DetailPredmetu.js

```diff
@@ -19,6 +19,20 @@ export var DetailPredmetuModal = React.createClass({
     return cache.get('get_studenti_zapisani_na_predmet', predmetKey, akademickyRok);
   },
 
+  renderInformacnyListPredmetu() {
+    var cache = new CacheRequester();
+    var {modalAkademickyRok, modalPredmetKey} = this.props.query;
+
+    var data = cache.get('get_informacny_list', modalPredmetKey, modalAkademickyRok);
```

Comment:
OK, v tom prípade to nemá cenu riešiť... aj `awaited_shell_exec` môže zostať ako je.